libclc/utils/libclc-remangler/LibclcRemangler.cpp

```diff
@@ -755,8 +741,8 @@ class TargetTypeReplacements {
       CloneTypeReplacements["unsigned long"] = "unsigned long long";
     }
 
-    // Make replaced char functions clones of either integer or long
-    // long variant
+    // Make replaced char functions clones of explicit signed or unsigned or
```

Comment:
```suggestion
    // Make replaced char functions clones of explicit signed int or unsigned int or
```
?

Reply:
No. It's referring to explicit signed/unsigned versions of char.

Reply:
Please, spell the whole type out. `long long` introduces a lot of confusion into this statement.

Reply:
I just had a second look at the code, and it seems the comment is actually copied from the long long changes, and this only deals with char. I'll updates the whole bit to make it make sense